statsdbinterface/database/extmodels.py

```diff
@@ -2,6 +2,7 @@
 from werkzeug.exceptions import NotFound
 from .core import db
 from .models import Game, GamePlayer, GameServer, GameWeapon
+from .models import Mutator as GMut, Mode as GMode
```

Comment:
Nothing bad about about splitting across multiple lines. I rename them because there're Mutator and Mode classes in this module already.